### DISCUSSION TITLE
unable to generate  3D inpainted mesh and demo video

### DISCUSSION THREAD

**Discussion by jkfor6** [thygate/stable-diffusion-webui-depthmap-script#182] [Q&A] [answered]
M1 Mac 2020, compute on CPU

- able to generate depth map.
- able to generate simple 3D mesh (not yet used in Blender, learning).
- unable to generate  3D inpainted mesh and demo video.

here's the message:

> DepthMap v0.3.11
> device: cpu
> Loading model weights from  ./models/leres/res101.pth
> initialize network with normal
> loading the model from ./models/pix2pix/latest_net_G.pth
> Computing depthmap(s) ..
>   0%|                                                     | 0/1 [00:00<?, ?it/s]
> 
> wholeImage being processed in : 1120
> Adjust factor is: 1.68
> Selecting patches ...
> Target resolution:  (2508, 3763, 3)
> Resulting depthmap resolution will be : (896, 1344)
> patches to process: 30
> 	 processing patch 0 / 29 | [  8   8 793 793]
> 	 processing patch 1 / 29 | [128   8 793 793]
> 	 processing patch 2 / 29 | [248   8 793 793]
> 	 processing patch 3 / 29 | [368   8 793 793]
> 	 processing patch 4 / 29 | [488   8 793 793]
> 	 processing patch 5 / 29 | [ 36 156 737 737]
> 	 processing patch 6 / 29 | [156 156 737 737]
> 	 processing patch 7 / 29 | [276 156 737 737]
> 	 processing patch 8 / 29 | [396 156 737 737]
> 	 processing patch 9 / 29 | [516 156 737 737]
> 	 processing patch 10 / 29 | [664  64 680 680]
> 	 processing patch 11 / 29 | [664 184 680 680]
> 	 processing patch 12 / 29 | [  9   9 547 547]
> 	 processing patch 13 / 29 | [  9 129 547 547]
> 	 processing patch 14 / 29 | [  9 249 547 547]
> 	 processing patch 15 / 29 | [129   9 547 547]
> 	 processing patch 16 / 29 | [249   9 547 547]
> 	 processing patch 17 / 29 | [369   9 547 547]
> 	 processing patch 18 / 29 | [489   9 547 547]
> 	 processing patch 19 / 29 | [609   9 547 547]
> 	 processing patch 20 / 29 | [729   9 547 547]
> 	 processing patch 21 / 29 | [156 397 491 491]
> 	 processing patch 22 / 29 | [277 397 491 491]
> 	 processing patch 23 / 29 | [397 397 491 491]
> 	 processing patch 24 / 29 | [517 397 491 491]
> 	 processing patch 25 / 29 | [637 397 491 491]
> 	 processing patch 26 / 29 | [905  64 434 434]
> 	 processing patch 27 / 29 | [905 184 434 434]
> 	 processing patch 28 / 29 | [  0 240 320 320]
> 	 processing patch 29 / 29 | [  0 360 320 320]
> 100%|████████████████████████████████████████████| 1/1 [02:28<00:00, 148.49s/it]
> Done.
> Running 3D Photo Inpainting .. 
> Loading edge model ..
> Loading depth model ..
> Loading rgb model ..
> All done.
> Error completing request
> Arguments: (0, <PIL.Image.Image image mode=RGB size=1344x896 at 0x28E3F0850>, None, '', '', 1, 0, 512, 512, False, True, False, False, 0, 1, False, 1, True, True, False, False, ['left-right', 'red-cyan-anaglyph'], 2.5, 'polylines_sharp', 0, True, True, False, False, False, 'u2net', False, 0, 2, False, None, True, False, True, False) {}
> Traceback (most recent call last):
>   File "/Users/jklee/stable-diffusion-webui/modules/call_queue.py", line 56, in f
>     res = list(func(*args, **kwargs))
>   File "/Users/jklee/stable-diffusion-webui/modules/call_queue.py", line 37, in f
>     res = func(*args, **kwargs)
>   File "/Users/jklee/stable-diffusion-webui/extensions/stable-diffusion-webui-depthmap-script/scripts/depthmap.py", line 1171, in run_generate
>     outputs, mesh_fi, meshsimple_fi = run_depthmap(
>   File "/Users/jklee/stable-diffusion-webui/extensions/stable-diffusion-webui-depthmap-script/scripts/depthmap.py", line 766, in run_depthmap
>     mesh_fi = run_3dphoto(device, inpaint_imgs, inpaint_depths, inputnames, outpath, fnExt, vid_ssaa, inpaint_vids)
>   File "/Users/jklee/stable-diffusion-webui/extensions/stable-diffusion-webui-depthmap-script/scripts/depthmap.py", line 843, in run_3dphoto
>     rgb_feat_weight = torch.load(colormodel_path, map_location=torch.device(device))
>   File "/Users/jklee/stable-diffusion-webui/modules/safe.py", line 106, in load
>     return load_with_extra(filename, extra_handler=global_extra_handler, *args, **kwargs)
>   File "/Users/jklee/stable-diffusion-webui/modules/safe.py", line 151, in load_with_extra
>     return unsafe_torch_load(filename, *args, **kwargs)
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/torch/serialization.py", line 713, in load
>     return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/torch/serialization.py", line 938, in _legacy_load
>     typed_storage._storage._set_from_file(
> RuntimeError: unexpected EOF, expected 10841829 more bytes. The file might be corrupted.
> 
> Traceback (most recent call last):
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/routes.py", line 394, in run_predict
>     output = await app.get_blocks().process_api(
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/blocks.py", line 1078, in process_api
>     data = self.postprocess_data(fn_index, result["prediction"], state)
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/blocks.py", line 1012, in postprocess_data
>     prediction_value = block.postprocess(prediction_value)
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/components.py", line 4307, in postprocess
>     "name": self.make_temp_copy_if_needed(y),
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/processing_utils.py", line 362, in make_temp_copy_if_needed
>     temp_dir = self.hash_file(file_path)
>   File "/Users/jklee/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/processing_utils.py", line 334, in hash_file
>     with open(file_path, "rb") as f:
> FileNotFoundError: [Errno 2] No such file or directory: "<div class='error'>RuntimeError: unexpected EOF, expected 10841829 more bytes. The file might be corrupted.</div><div class='performance'><p class='time'>Time taken: <wbr>2m 33.49s</p></div>"

please help.

**Comment by thygate** [comment, marked as answer]
seems like one of the inpainting models is corrupted, delete the models in models/3dphoto, and then try again.

I assume the download got cut off last time as the filesize is too small.

**Reply by jkfor6** [reply]
seems working! generating inpainted mesh now, let's see the result

**Reply by jkfor6** [reply]
it's done! thank you so much!
